core/src/main/java/io/smallrye/openapi/api/OpenApiConfigImpl.java

```diff
@@ -20,6 +20,17 @@
  */
 public class OpenApiConfigImpl implements OpenApiConfig {
 
+    private static final String VERSION = OASConfig.EXTENSIONS_PREFIX + "openapi";
+    private static final String INFO_TITLE = OASConfig.EXTENSIONS_PREFIX + "info.title";
+    private static final String INFO_VERSION = OASConfig.EXTENSIONS_PREFIX + "info.version";
+    private static final String INFO_DESCRIPTION = OASConfig.EXTENSIONS_PREFIX + "info.description";
+    private static final String INFO_TERMS = OASConfig.EXTENSIONS_PREFIX + "info.termsOfService";
+    private static final String INFO_CONTACT_EMAIL = OASConfig.EXTENSIONS_PREFIX + "info.contact.email";
+    private static final String INFO_CONTACT_NAME = OASConfig.EXTENSIONS_PREFIX + "info.contact.name";
+    private static final String INFO_CONTACT_URL = OASConfig.EXTENSIONS_PREFIX + "info.contact.url";
+    private static final String INFO_LICENSE_NAME = OASConfig.EXTENSIONS_PREFIX + "info.license.name";
+    private static final String INFO_LICENSE_URL = OASConfig.EXTENSIONS_PREFIX + "info.license.url";
```

Comment:
These should probably include `smallrye` in the name. Take a look in `io.smallrye.openapi.api.constants.OpenApiConstants` for the others.

Reply:
Ok cool thanks. I'll actually move the constants to `io.smallrye.openapi.api.constants.OpenApiConstants` as well.